samples/deviceCodeSample/src/main/java/deviceCodeFlowMain.java

```diff
@@ -0,0 +1,45 @@
+import com.azure.identity.DeviceCodeCredential;
+import com.azure.identity.DeviceCodeCredentialBuilder;
+import com.microsoft.graph.authentication.TokenCredentialAuthProvider;
+import com.microsoft.graph.httpcore.HttpClients;
+import okhttp3.*;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+
+public class deviceCodeFlowMain {
```

Comment:
Out of curiosity, what's the reason for having class name in lowerCamelCase?

Reply:
fixed